src/query/ast/tests/it/testdata/statement.txt

```diff
@@ -5926,6 +5970,51 @@ Query(
 )
 
 
+---------- Input ----------
+select 'stringwith''quote'''
+---------- Output ---------
+SELECT 'stringwith'quote''
```

Comment:
here is a bad case about string's quote (not introduced in this PR), maybe the string literal needs some escaping